test/float8/test_base.py

```diff
@@ -231,15 +231,6 @@ def test_linear(
         linear_dtype: torch.dtype,
         linear_bias: bool,
     ):
-        if not emulate:
-            if not torch.cuda.is_available():
-                warnings.warn("CUDA not available")
-                pytest.skip()
-            elif torch.cuda.get_device_capability() < (9, 0):
```

Comment:
this line made this test skip in CI, not intended, removing.  Similar cases in the rest of this file.